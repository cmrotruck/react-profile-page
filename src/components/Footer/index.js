import React from "react";

const Footer = () => {
  return (
    <footer className="">
      <img
        className="footerLogo"
        src={require("../../assets/footer/27-277988_linkedin-logo-png-branco.png")}
        alt="small linkedIn logo"
      ></img>
      <img
        className="footerLogo"
        src={require("../../assets/footer/GitHub-Mark-Light-120px-plus.png")}
        alt="small Github logo"
      ></img>
    </footer>
  );
};

export default Footer;
