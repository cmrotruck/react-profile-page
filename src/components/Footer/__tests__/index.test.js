import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Footer from "..";

afterEach(cleanup);

describe("Nav component", () => {
  it("renders correctly", () => {
    render(<Footer />);
  });

  it("matches snapshot", () => {
    const { asFragment } = render(<Footer />);

    expect(asFragment()).toMatchSnapshot();
  });

  it("linkedIn logo renders", () => {
    render(
      <img src="./assets/footer/LI-ln-Bug.png" alt="linked in logo small"></img>
    );
  });
});
