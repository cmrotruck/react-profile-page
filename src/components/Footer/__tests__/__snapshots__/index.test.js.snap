// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component matches snapshot 1`] = `
<DocumentFragment>
  <footer>
    <img
      alt="linked in logo small"
      src="./assets/footer/LI-ln-Bug.png"
    />
  </footer>
</DocumentFragment>
`;
